registry-login/src/context.ts

```diff
@@ -0,0 +1,41 @@
+import * as core from '@actions/core'
+
+export interface Inputs {
+  registryHostname: string
+  registryNamespace: string
```

Comment:
可以不用这些前缀了，因为当前的上下文是在一个 registry 中，它里面就隐含有这些信息
```suggestion
  hostname: string
  namespace: string
```